features/bsm/rest/src/main/java/org/opennms/web/rest/v2/BusinessServiceRestService.java

```diff
@@ -139,11 +139,11 @@ public Response detachIpService(@PathParam("id") final Long serviceId,
 
     @GET
     @Path("{id}/ip-service/operational-status/{ipServiceId}")
-    public Response getOperationStatusForIPServiceById(@PathParam("id") Integer ipServiceId) {
+    public Response getOperationStatusForIPServiceById(@PathParam("id") Integer businessServiceId, @PathParam("ipServiceId") Integer ipServiceId) {
```

Comment:
Do you refer to the "{id}/ip-services" in the path?
That is used this way in 6 other methods in the class and makes sense in the BusinessServiceRestService class.